src/other/niu_mini/niu_mini.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name":"niu_mini",
+  "vendorId":"0xFEED",
+  "productId":"0x6060",
```

Comment:
Please change the vendor and product IDs

Reply:
Is there general guidance on allocating vendor and product IDs for VIA/QMK, or do I just find values that haven't been used and checked in yet?

Reply:
See here: https://caniusevia.com/docs/configuring_qmk#change-vendor_id-product_id

I suggest pick a unique one for the vendor, and start numbering product at 0x0001

Reply:
updated